packages/components/src/services/HomePageService.ts

```diff
@@ -0,0 +1,23 @@
+import { HttpService } from '../utils/httpService';
+import { HomePageInitSchema } from '../schemas';
+
+const apiUrl = 'https://breakup-app-api.herokuapp.com/api';
```

Comment:
same URL used here, put it HttpService.ts

Reply:
will be added to env variable in further PR's